bluetoothle/src/main/java/com/sample/android/bluetoothle/kotlin/MainActivity.kt

```diff
@@ -28,8 +28,19 @@ class MainActivity : AppCompatActivity() {
         // displays a dialog requesting user permission to enable Bluetooth.
         if (bluetoothAdapter == null || !bluetoothAdapter.isEnabled) {
             val enableBtIntent = Intent(BluetoothAdapter.ACTION_REQUEST_ENABLE)
-            startActivityForResult(enableBtIntent, REQUEST_ENABLE_BT)
+            getResult.launch(enableBtIntent)            
```

Comment:
Also add a comment:

// Launch the request with the ActivityResultContracts.StartActivityForResult